docs/production-deployment/cloud/nexus/getting-started.mdx

```diff
@@ -15,9 +15,11 @@ keywords:
 
 ---
 
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
 Temporal Nexus Public Preview is available within your Temporal Cloud account, with support in the [Temporal Go SDK](/develop/go).
 
-Calls across existing Namespaces can be enabled by creating a Nexus Endpoint in the Nexus API Registry, creating a Nexus Service in a Worker in the handler Namespace, and then using the Nexus Service from a caller Workflow in a different Namespace.
+Calls across existing Namespaces can be enabled by creating a Nexus Endpoint in the Nexus Registry, creating a Nexus Service in a Worker in the handler Namespace, and then using the Nexus Service from a caller Workflow in a different Namespace.
```

Comment:
This is a very long sentence with a lot of thoughts in it. Break it down into bullet points?

Enable calls across (existing) Namespaces by:
- Creating a Nexus Endpoint in the Nexus Registry
- Creating a Nexus Service in a Worker within a handler Namespace
- Calling the Nexus Service from a Workflow in a different Namespace

Reply:
fixed and consolidated the bullet list below